readme.md

```diff
@@ -0,0 +1,26 @@
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+- [👥 Authors](#authors)
+
+
+
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+
+
+**[Hello Microverse]** is my first mniproject from module 1. This project serves to familiarize students with Git, Github,gitflow and linters.
+
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+👤 Andrew McCollin
+
+- GitHub: (https://github.com/Drew246)
+- LinkedIn: [www.linkedin.com/in/andrew-mccollin)
+

```

Comment:
- [ ] You have submitted a project that is not professional and does not include all the headers/sections present in [readme template](https://github.com/microverseinc/readme-template)📖
 
- Note that the placeholders in README file that are meant to guide you on the contents of each of the sections. They are not meant to instruct users on how to contribute to the code so please ensure that you remove them.
- [ ] Please ensure that you professionally document the README file to reflect the project 😄

- **Your README**

![image](https://github.com/Drew246/firstMicroProject/assets/68823331/fe77b4a9-1fd6-4a08-814a-0cda885cfea6)

-  **Microverse README**

![image](https://github.com/Drew246/firstMicroProject/assets/68823331/a853b0ce-fbd9-48a0-b952-bab1c999a897)

Reply:
Hi Christopher. As you can imagine this is a bit confusing for me at the moment. The exercise instructions said to only include sections that are relevant to the project. So I assumed that I could delete most of the other stuff since they are not relevant. let me know if I should still fix it.

Reply:
Hello @Drew246 , Kindly note that maintaining a professional tone throughout the README, including the header section, is a requirement. In instances where the content does not currently apply to the project, it can be replaced with 'N/A'.

For further guidance and to ensure adherence to the expected standards, I recommend consulting the [Microverse Readme Template](https://github.com/microverseinc/readme-template). 👍🏿